staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -429,25 +429,26 @@ func decodeContinue(continueValue, keyPrefix string) (fromKey string, rv int64,
 		return "", 0, fmt.Errorf("continue key is not valid: %v", err)
 	}
 	switch c.APIVersion {
-	case "v1alpha1":
+	case "meta.k8s.io/v1":
```

Comment:
we have constants for this

Reply:
These are not publicly accessible - even if we have constants I think we reserve the right to change this.